auth.php

```diff
@@ -104,6 +104,19 @@ public function __construct() {
         $this->defaultidp = auth_saml2_get_default_idp();
     }
 
+    public static function saml2_authproc_filters_hook()
+    {
+        if (function_exists('get_plugins_with_function')) {
```

Comment:
is this if needed?

Reply:
you're right this is not needed. removed it.